lib/events_collection/orchestrator_event.rb

```diff
@@ -0,0 +1,13 @@
+module CommonEvents
```

Comment:
This file can be removed. This is from when we thought the module would create objects to wrap events returned by the api. We aren't going to do that anymore.